opencv-examples/calibrate.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+import numpy as np
+import cv2
+import os
+import argparse
+import yaml
+import pickle
+from glob import glob
+
+# Taken from https://github.com/smidm/video2calibration and adopted to python 3
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Calibrate camera using a video of a chessboard or a sequence of images.')
+    parser.add_argument('input', help='input video file or glob mask')
+    parser.add_argument('out', help='output calibration yaml file')
+    parser.add_argument('--debug-dir', help='path to directory where images with detected chessboard will be written',
+                        default=None)
+    parser.add_argument('-c', '--corners', help='output corners file', default=None)
+    parser.add_argument('-fs', '--framestep', help='use every nth frame in the video', default=20, type=int)
+    # parser.add_argument('--figure', help='saved visualization name', default=None)
+    args = parser.parse_args()
+
+    if '*' in args.input:
+        source = glob(args.input)
+    else:
+        source = cv2.VideoCapture(args.input)
+    # square_size = float(args.get('--square_size', 1.0))
+
+    pattern_size = (9, 6)
```

Comment:
I don't even ask, why the numbers in this script have the values they have :laughing:

Reply:
9, 6 is the amount of inner corners in the checkerboard pattern. One per axis ;) The 1.0 is -- i think -- the aspect ratio of the squares.